pages/[username]/index.tsx

```diff
@@ -1,55 +1,20 @@
-import {
-  AccountWithSensorsType,
-  getAccountDataByUsername,
-} from "@lib/requests/getAccountDataByUsername";
 import { GetServerSideProps } from "next";
 import { FC } from "react";
-import { UserInfoWithData } from "@components/UserInfoHeader/withData";
-import { SensorsList } from "@components/SensorsList";
 
+// eslint-disable-next-line @typescript-eslint/require-await
 export const getServerSideProps: GetServerSideProps = async context => {
-  try {
-    const username = context.query.username;
-    if (!username || Array.isArray(username)) return { notFound: true };
-    const accountData = await getAccountDataByUsername(username);
-    return {
-      props: {
-        account: { ...accountData },
-        error: null,
-      },
-    };
-  } catch (error) {
-    console.error(error);
-    return { notFound: true };
-  }
-};
-
-interface AccountSensorsPagePropType {
-  account: AccountWithSensorsType;
-}
+  const username = context.query.username;
+  if (!username || Array.isArray(username)) return { notFound: true };
 
-const AccountSensorsPage: FC<AccountSensorsPagePropType> = ({ account }) => {
-  return (
-    <>
-      <UserInfoWithData routeAccount={account} activeTab='sensors' />
-      <div
-        id='tab-content'
-        role='tabpanel'
-        className={[
-          "container max-w-8xl mx-auto px-4 pt-8 pb-24 min-h-[500px]",
-          account.sensors.length === 0
-            ? "flex place-items-center place-content-center"
-            : "",
-        ].join(" ")}
-      >
-        {account.sensors.length > 0 ? (
-          <SensorsList sensors={account.sensors} />
-        ) : (
-          <p>Keine Sensoren vorhanden</p>
-        )}
-      </div>
-    </>
-  );
+  return {
+    redirect: {
+      destination: `/${username}/sensors`,
+      permanent: true,
```

Comment:
Does the permanent redirect make sense here? Based on @Esshahn's comment earlier today `username/index` might actually turn into a real page one day.

Reply:
But it will be completely new content. Therefore it isn't a temporary redirect either. It isn't a previously available page that will be reinserted later on. I wouldn't bet my life on this though. I'm really not sure.

Reply:
I did some research and it seems you are right! This particular page will potentially be used in the future and we should therefore not do a permanent redirect. I will change that to be a temporary redirect.